packages/compat/src/default-pipeline.ts

```diff
@@ -12,7 +12,9 @@ export default function defaultPipeline<PackagerOptions>(
   packager: Packager<PackagerOptions>,
   options?: PipelineOptions<PackagerOptions>
 ): Tree {
+
   let addons;
+  AppBuilder.activateMacroConfig();
```

Comment:
@ef4 we should find some time to chat about this. We need to figure out an appropriate solution.